packages/web/docs/src/pages/features/sso-oidc-provider.mdx

```diff
@@ -55,3 +55,180 @@ Provider. Share this link with your users or add it to your SSO provider dashboa
 people will be redirected to it.
 
 E.g. `https://app.graphql-hive.com/auth/oidc?id=xxxxxxxx-1234-1234-1234-xxxxxxxxxxxx`
+
+## Configuring with Okta
+
+First we need to have an Okta account, so you can register for an account from
```

Comment:
I think we can skip the trial part. If somebody is interested in connecting Okta to Hive, they probably have Okta account already.

Reply:
Yeah, agree